tasks/main.yml

```diff
@@ -87,52 +101,36 @@
   changed_when: '"imported" in caddy_pgp_key.stdout'
   tags: skip_ansible_lint
 
-- name: Download Caddy signature
-  get_url:
-    url: "{{ caddy_sig_url }}"
-    dest: "{{ caddy_home }}/caddy.tar.gz.asc"
-    timeout: 60
-    force: true
-    force_basic_auth: "{{ caddy_license != 'personal' }}"
-  when: caddy_pgp_verify_signatures and (caddy_binary_cache.changed or caddy_download.changed)
-  tags: skip_ansible_lint
-
-- name: Verify Caddy signature
-  command: >
-    gpg
-      --verify {{ caddy_home }}/caddy.tar.gz.asc
-      {{ caddy_home }}/caddy.tar.gz
-  when: caddy_pgp_verify_signatures and (caddy_binary_cache.changed or caddy_download.changed)
-  tags: skip_ansible_lint
-
-- name: Extract Caddy
-  unarchive:
-    src: "{{ caddy_home }}/caddy.tar.gz"
-    dest: "{{ caddy_home }}"
-    copy: false
-    owner: "{{ caddy_user }}"
-  when: caddy_binary_cache.changed
-  tags: skip_ansible_lint
-
-- name: Extract Caddy
-  unarchive:
-    src: "{{ caddy_home }}/caddy.tar.gz"
-    dest: "{{ caddy_home }}"
-    creates: "{{ caddy_home }}/caddy"
-    copy: false
-    owner: "{{ caddy_user }}"
+# - name: Download Caddy signature
```

Comment:
If this doesn't exist anymore, then we should fully remove all the code.

Reply:
Yes I have fully removed it for now - there are no signatures available for the builds currently.